sct.py

```diff
@@ -1351,5 +1356,33 @@ def clean_aws_mocks(test_id: str | None, all_mocks: bool, verbose: bool, dry_run
     AwsMock.clean(test_id=test_id, all_mocks=all_mocks, verbose=verbose, dry_run=dry_run)
 
 
+@cli.command("generate-pt-report", help="Generate parallel timelines representation for the SCT test events")
+@click.option("-t", "--test-id", help="Test ID to search in sct-results")
+@click.option("-d", "--logdir", help="Directory with sct-results folder")
+def generate_parallel_timelines_report(logdir: str | None, test_id: str | None) -> None:
+    event_log_file = "raw_events.log"
+    if not test_id:
+        test_id = os.getenv("SCT_TEST_ID")
+    if not logdir:
+        logdir = os.environ.get('HOME')
+    LOGGER.debug("Searching for the required test run directory in %s...", logdir)
+    testrun_dir = get_testrun_dir(os.path.join(logdir, "sct-results"), test_id)
+    if not testrun_dir:
+        click.secho(message=f"Couldn't find directory for the required test run in '{logdir}'! Aborting...", fg="red")
+        sys.exit(1)
+    LOGGER.info("Found the test run directory '%s'", testrun_dir)
+    cmd = f"find {testrun_dir} -name {event_log_file}"
+    LOGGER.debug("Searching for the %s in %s...", event_log_file, testrun_dir)
+    result = LOCALRUNNER.run(cmd=cmd, ignore_status=True)
+    if result.exited == 0 and result.stdout:
```

Comment:
You can use `result.ok`

Reply:
Removed this since I don't use `LOCALRUNNER.run()` anymore